docs/src/index.md

```diff
@@ -64,3 +64,50 @@ Ferrite.debug_mode(enable=false)
 ```
 
 also followed by restarting the Julia process.
+
+## Performance Analysis
```

Comment:
Could (should?) move to a new page in devdocs.

Reply:
Give me 5 minutes.